ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/AccessList.java

```diff
@@ -21,14 +21,11 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
 
-import org.apache.tuweni.bytes.Bytes32;
+public class AccessList extends ArrayList<AccessListEntry> {
```

Comment:
What do we gain by this specialization? Directly extending collections objects feels weird.

Reply:
I was trying to do a type alias as is common in other languages and realized that since Java doesn't have those, this was the easiest way to accomplish the same thing. The main benefit is just less indirect access to iteration. I suppose it doesn't look so idiomatic in java though.

Reply:
Subclassing collections should be reserved for collections type extensions,  Like a sorted list, queue, or such.  This would also hard lock us into a specific kind of list.  

Idiomatic java just uses var or diamonds after initial decleration.